swift/Tests/Addresses/BitcoinAddressTests.swift

```diff
@@ -137,9 +137,22 @@ class BitcoinAddressTests: XCTestCase {
         XCTAssertEqual(P2PKHPrefix.ravencoin.rawValue, BitcoinAddress(string: addressString)?.prefix)
         XCTAssertTrue(BitcoinAddress.isValidString(string: addressString),
                       "'\(addressString)' should be a valid Ravencoin address")
+
+        let addressString2 = "rPWwn5h4QFZNaz1XmY39rc73sdYGGDdmq1"
+
+        XCTAssertEqual(P2SHPrefix.ravencoin.rawValue, BitcoinAddress(string: addressString2)?.prefix)
+        XCTAssertTrue(BitcoinAddress.isValidString(string: addressString2),
+                      "'\(addressString2)' should be a valid Ravencoin address")
+
+        // testnet address
+        let addressString3 = "mwJAu1BWcRSQhepZ71wiGoSwsD6hnB5B7G"
+
+        XCTAssertTrue(BitcoinAddress.isValidString(string: addressString3),
+                       "'\(addressString3)' should be a valid Ravencoin testnet address")
```

Comment:
testnet address is valid in iOS test as `BitcoinAddress.isValidString()`eventually calls `Address::isValid(s)` in `Base58Address.h`, where it doesn't check prefix.
if we would like to deem testnet address invalid, we need to call `static bool isValid(const std::string& string, const std::vector<Data>& validPrefixes)`, what do you reckon @hewigovens @vikmeup ?

Reply:
try `CoinType.validate`

Reply:
> try `CoinType.validate`

Thanks, done in 8355d4c9d3f5360e5968ab95a42dafa6a9b78d48